[Bug]: Merge CI, test_all_non_excluded_pipelines_have_tests - WanDMDPipeline missing test settings

### Your current environment

ci env

### Your code version

vllm: CI
vllm-omni: 072bfc0

### 🐛 Describe the bug

**Buildkite build:** https://buildkite.com/vllm/vllm-omni/builds/14011
**Failure kind:** pytest FAILED
**Test node:** \tests/model_tests/diffusion/test_alignment.py::test_all_non_excluded_pipelines_have_tests
**Log reason:**
PR #4820 added new pipeline `WanDMDPipeline` in `vllm_omni/diffusion/registry.py` but did not register test settings in `tests/model_tests/diffusion/model_settings.py`, so `test_all_non_excluded_pipelines_have_tests` fails.
**Error log excerpt:**
FAILED tests/model_tests/diffusion/test_alignment.py::test_all_non_excluded_pipelines_have_tests - AssertionError: Pipelines missing test settings: {'WanDMDPipeline'}
assert 1 == 0
+  where 1 = len({'WanDMDPipeline'})
= 1 failed, 2491 passed, 12 skipped, 988 deselected, 37 warnings in 325.49s (0:05:25) =

### Before submitting a new issue...

- [x] Make sure you already searched for relevant issues, and asked the chatbot living at the bottom right corner of the [documentation page](https://vllm-omni.readthedocs.io), which can answer lots of frequently asked questions.

--- a/tests/model_tests/diffusion/test_alignment.py
+++ b/tests/model_tests/diffusion/test_alignment.py
@@ -26,6 +26,7 @@ EXCLUDED_MODELS = [
     "ZImagePipeline",
     "OvisImagePipeline",
     "WanPipeline",
+    "WanDMDPipeline",
     "WanVACEPipeline",
     "LTX2TwoStagePipeline",
     "LTX2DistilledOneStagePipeline",

